CHANGELOG.md

```diff
@@ -27,6 +27,21 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - Changed `COMPATIBLE_VERSION_REQ` to ">=0.24.0-alpha.1,<0.25.0".
   - Added migration function `migrate_0_23_0_to_0_24_0_op_log`. This function
     performs a migration to change the key and value of `Oplog`.
+- Added minimal mode to improve behavior in remote configuration mode.
+  - If local configuration is not used when running Giganto, it will run in
+    minimal mode, with only the GraphQL server running. Retain Task, Peer Server
+    Task, Ingest Task, Publish Task, and DB migration are not executed.
```

Comment:
I think it could be minor, but I think the we should use lower case in "Retain Task, Peer Server Task, Ingest Task, Publish Task" except the R in the start of the sentence.

Reply:
And I don't think we put period at the end of the main commit message.

Reply:
done